R/angles.R

```diff
@@ -26,43 +23,118 @@ calculate.angles.radial <- function(v, tree, spread, total.angle) {
         num.children <- length(child.ids);
 
         if (num.children > 0) {
-            parent.angle <- angles[current.node.id];
+            parent.angle <- angles[tree$tip == current.node.id];
```

Comment:
Assuming you made this change to support arbitrary node IDs? (Rather than auto-generated, ordered IDs that can be indexed directly)

Reply:
no, I initially change it because I thought a named list for `angles` might work better but i was worried it might get messed up since the `tree$tip` are also numeric. So the added code is actually reverting to original.